packages/test-utils/console.d.ts

```diff
@@ -0,0 +1,3 @@
+export function silenceConsoleErrors(): {
+    [Symbol.dispose](): void;
+};
```

Comment:
Just committing the generated .d.ts files here - setting up something with project references doesn't work nicely with `tsx`, and honestly it's not worth the hassle.